tasks/system_probe.py

```diff
@@ -1370,3 +1370,20 @@ def save_test_dockers(ctx, output_dir, arch, windows=is_windows):
         output_path = image.translate(str.maketrans('', '', string.punctuation))
         ctx.run(f"docker pull --platform linux/{arch} {image}")
         ctx.run(f"docker save {image} > {os.path.join(output_dir, output_path)}.tar")
+
+
+@task
+def test_microvms(ctx, security_groups="", subnets="", instance_type_x86="", instance_type_arm="", destroy=False):
```

Comment:
I think convention is to use `None` for default values of arguments, rather than empty string.